odl/solvers/functional/functional.py

```diff
@@ -225,6 +225,48 @@ def translated(self, shift):
         """
         return FunctionalTranslation(self, shift)
 
+    def bregman(self, point, subgrad=None):
+        """Return the Bregman distance functional.
+
+        Parameters
+        ----------
+        point : element of ``functional.domain``
+            Point from which to define the Bregman distance.
+        subgrad : `Operator`, optional
```

Comment:
When I define the Bregman distance, I normally would like to give a specific subgradient and not a subgradient operator that would compute one.